pyproject.toml

```diff
@@ -42,6 +42,8 @@ dependencies = [
     "markdownify >= 0.11",
     "deprecation >= 2.1",
     "packaging",
+    "tabulate >= 0.9.0",
+    "rich >= 13.7.0",
```

Comment:
Having both rich and tabulate feels like an overkill, fairly certain they won't play nice with each other (use same colors etc)

Reply:
Seems to work fine:
![image](https://github.com/mercedes-benz/odxtools/assets/142314987/13b124c2-d074-47c9-a21c-e6db3100e11e)
Nevertheless, I'm open to get rid of tabulate and switch to rich tables in the future